api/SVGElement.json

```diff
@@ -96,6 +96,53 @@
           }
         }
       },
+      "className": {
```

Comment:
Where else in BCD do we already have a "className" entry?

Reply:
`api.Element.className` exists but this one is different. It returns an object instead of a string, and thus is deprecated.

Reply:
Oh, so both exist. This will shadow the one on Element, so I guess deprecating simply means "don't use" in this case then.